code/osapi/osapi.h

```diff
@@ -139,8 +139,6 @@ namespace os
 
 		uint32_t depth_size;
 		uint32_t stencil_size;
-
-		uint32_t multi_samples; //!< The amount of multi-sampling, use 0 for no multi-sampling
```

Comment:
IMO you shouldn't remove the option from the viewports API since multi-sampling is still a valid option and may even be enabled again later if we ever find a fix for this bug. You should just set `multi_samples` to `0` and ignore the config value.